jenkins-scripts/lib/dependencies_archive.sh

```diff
@@ -499,17 +487,19 @@ if [[ ${DISTRO} != 'xenial' ]]; then
                       qml-module-qtquick-window2 \\
                       qml-module-qtquick2 \\
                       qtquickcontrols2-5-dev"
-fi
 
 IGN_GUI_DEPENDENCIES="${IGN_GUI_NO_IGN_DEPENDENCIES} \\
+                      libignition-cmake2-dev \\
+                      libignition-common3-dev \\
+                      libignition-math6-dev \\
+                      libignition-msgs3-dev \\
+                      libignition-plugin-dev \\
+                      libignition-rendering-dev \\
                       libignition-tools-dev"
 
-if [[ ${DISTRO} != 'xenial' ]]; then
+if [[ -n "${IGN_GUI_MAJOR_VERSION}" && ${IGN_GUI_MAJOR_VERSION} -eq 0 ]]; then
```

Comment:
GUI 0 and Rendering 2 are EOL, so this block can be removed

Reply:
done 0309df7